api/comments.rb

```diff
@@ -1,5 +1,5 @@
 get "#{APIPREFIX}/comments/:comment_id" do |comment_id|
-  comment.to_hash(recursive: bool_recursive).to_json
+  comment(nil, true).to_hash(recursive: bool_recursive).to_json
```

Comment:
I think I'd put this back the way it was.  I don't understand why the nil is being passed when I don't see any uses of that first parameter.

Instead, I'd do something like the following:
```
comment_hash = comment.to_hash(recursive: bool_recursive)
verify_or_fix_cached_comment_count(comment_hash)
comment_hash.to_json
```
Then add the following method:
```
# Verifies that the cached comment count is accurate.  Otherwise, it fixes it by updating it.
def verify_or_fix_cached_comment_count(comment_hash)
```